cloudfront/wellcomelibrary.org/terraform/dns.tf

```diff
@@ -1,9 +1,45 @@
-resource "aws_route53_record" "prod" {
+resource "aws_route53_record" "prod-internal" {
   zone_id = data.aws_route53_zone.zone.id
   name    = "wellcomelibrary.org"
   type    = "A"
   records = ["195.143.129.236"]
-  ttl     = "300"
+  ttl     = "60"
+
+  weighted_routing_policy {
+    weight = 0
+  }
+
+  set_identifier = "internal"
+
+  provider = aws.dns
+}
+
+resource "aws_route53_record" "prod-cloudfront" {
+  zone_id = data.aws_route53_zone.zone.id
+  name    = "wellcomelibrary.org"
+  type    = "A"
+
+  weighted_routing_policy {
+    weight = 50
+  }
+
+  alias {
+    name                   = "d2cksn8op995w4.cloudfront.net"
+    evaluate_target_health = false
+    zone_id                = "Z2FDTNDATAQYW2"
```

Comment:
Where does this Zone ID come from?

Reply:
Good catch - I copy/pasted this to satisfy a terraform import and didn't update it. I can just pull this from the prod CloudFront distro module 👍

Reply:
Oh, this is an interesting one actually - this value is the same for all cloudfront distros (added a comment with link to docs).